pom.xml

```diff
@@ -56,7 +56,7 @@
 
   <properties>
     <javaVersion>8</javaVersion>
-    <mavenVersion>3.2.5</mavenVersion>
+    <mavenVersion>3.6.3</mavenVersion>
     <slf4jVersion>1.7.36</slf4jVersion>
     <project.build.outputTimestamp>2023-04-15T15:52:39Z</project.build.outputTimestamp>
     <junitVersion>5.10.2</junitVersion>
```

Comment:
We cen drop `junitVersion` - we have dependencyManagement in parent

Reply:
Will do